src/renderers/webgl/WebGLProgram.js

```diff
@@ -340,6 +341,9 @@ function WebGLProgram( renderer, extensions, code, material, shader, parameters,
 
 	var prefixVertex, prefixFragment;
 
+	var renderTarget = renderer.getRenderTarget();
+	var numMultiviewViews = renderTarget instanceof WebGLMultiviewRenderTarget ? renderTarget.numViews : 0;
```

Comment:
We try not to use `instanceof`:

```js
var numMultiviewViews = renderTarget.isWebGLMultiviewRenderTarget ? renderTarget.numViews : 0;
```

Reply:
Yep I forgot that one, I was using `isWebGL..` on the rest of the code already. Thanks